flocker/acceptance/integration/test_platform.py

```diff
@@ -0,0 +1,47 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for integration with the host operating system which runs Flocker.
+"""
+
+from ..testtools import require_cluster
+
+from twisted.python.filepath import FilePath
+from twisted.trial.unittest import TestCase
+
+
+class SyslogTests(TestCase):
+    """
+    Tests for Flocker's integration with syslog.
+    """
+    def _assert_not_logged(self, cluster, fragment):
+        """
+        Assert that the given fragment of a log line does not appear in the
+        ``/var/log/messages`` file on the control node of the cluster.
+        """
+        getting = cluster.get_file(
+            cluster.control_node,
+            FilePath(b"/var/log/messages"),
+        )
+
+        def got_messages(path):
+            with path.open() as messages:
+                for line in messages:
+                    self.assertNotIn(fragment, line)
+
+        getting.addCallback(got_messages)
+        return getting
+
+    @require_cluster(1)
+    def test_flocker_control_not_logged(self, cluster):
+        """
+        Log messages from ``flocker-control`` do not appear in syslog.
+        """
+        return self._assert_not_logged(cluster, b"flocker:controlservice:")
```

Comment:
:question: Why don't you have to exercise the system? I guess it's because `require_cluster` does stuff that we know for a fact logs things? 

I think the test would more clearly communicate intent if they did a log-triggering action, or at least had a comment explaining why we know something has been logged.